### PR TITLE
(ruby.portable) Migrate package, update code and metadata, add AU

### PR DESCRIPTION
**Pull request by TheCakeIsNaOH**
## Description

This migrates in ruby.portable. 

The metadata is updated in accordance with the ruby package's metadata, the package is switched to embedded, and the install script code style is updated.

An AU script is also added, 

## Motivation and Context

Part of #1757 
Should not be merged until #1786 is merged

## How Has this Been Tested?

Ran `update.ps1` to validate that it creates packages for the various minor releases (2.4 and newer).

Installed and uninstalled packages in test env to validate that they work.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [x] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](https://github.com/ferventcoder/chocolatey-packages/tree/master/manual/ruby.portable)
- [Open Issues](https://github.com/ferventcoder/chocolatey-packages/issues) *Add the different issues underneath, and tick those that are fixed in this PR*
- [x] [File removal PR](https://github.com/ferventcoder/chocolatey-packages/pull/335)
- [x] The [migration guidelines](https://github.com/chocolatey-community/chocolatey-packages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/ruby.portable/README.md
Comment (AdmiringWorm) [inline automatic/ruby.portable/README.md:4]: Ruby is a dynamic, open source programming language focusing on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.

File: automatic/ruby.portable/README.md
Comment (AdmiringWorm) [inline automatic/ruby.portable/README.md:8]:
```suggestion
- To install ruby development kit ruby installer provides `ridk` command. It provides an easy way to install msys2 via `ridk install 1`, however, the installation is interactive. To accomplish unattended install, use [msys2](https://chocolatey.org/packages/msys2) package.
```

File: automatic/ruby.portable/legal/LICENSE.txt
Comment (AdmiringWorm) [inline automatic/ruby.portable/legal/LICENSE.txt:1]: This license looks a bit outdated, is this really the latest license available?

File: automatic/ruby.portable/tools/chocolateyInstall.ps1
Comment (AdmiringWorm) [inline automatic/ruby.portable/tools/chocolateyInstall.ps1:8]: Instead of changing the package name to a fixed value, can you please change it to

```suggestion
  packageName = $env:ChocolateyPackageName
```

File: automatic/ruby.portable/update.ps1
Comment (AdmiringWorm) [inline automatic/ruby.portable/update.ps1:8]: Don't replace the package name, instead use the requested change for the install script.

File: automatic/ruby.portable/update.ps1
Comment (AdmiringWorm) [inline automatic/ruby.portable/update.ps1:10]:
```suggestion
            "(?i)(^\s*file\s*=\s*`"[$]toolsPath\\).*"   = "`${1}$($Latest.FileName32)`""
            "(?i)(^\s*file64\s*=\s*`"[$]toolsPath\\).*" = "`${1}$($Latest.FileName64)`""
```

File: automatic/ruby.portable/update.ps1
Comment (AdmiringWorm) [inline automatic/ruby.portable/update.ps1:65]:
```suggestion
update -ChecksumFor none
```

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
Quite a few things need to be fixed up, and in addition to those can you please fix up the formatting for the PowerShell and Nuspec files?

Indentation should be equal to 2 spaces, no trailing spaces at the end of files and a newline at the end of each file.

See the `.editorconfig` file: https://github.com/chocolatey-community/chocolatey-packages/blob/master/.editorconfig

File: automatic/ruby.portable/legal/LICENSE.txt
Comment (TheCakeIsNaOH) [reply automatic/ruby.portable/legal/LICENSE.txt]: Just the year is now 2020, no other changes. The URL in the verification was outdated, I've fixed that up, and will fix it up for the other ruby packages as well.

**Comment by TheCakeIsNaOH** [comment]
@AdmiringWorm Updated this PR, thanks for the review. I think the formatting should be ok now?

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
Just a single thing that I missed previously, other than that we just need to figure out who will be the code owner/responsible for this package going forward before we can merge the changes in.

File: automatic/ruby.portable/ruby.portable.nuspec
Comment (TheCakeIsNaOH) [reply automatic/ruby.portable/ruby.portable.nuspec:4]: Done

**Review by AdmiringWorm — APPROVED** [review]
LGTM. But as I have mentioned previously we'll need to figure out who will be responsible for the package before we pull the changes in.

**Comment by AdmiringWorm** [comment]
The Code Owners has been decided upon, as such I have merged this PR now.

@TheCakeIsNaOH thank you for updating the package and getting it here.

**Pull request merged**
